<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalMnemonicEditor</class>
 <widget class="QDialog" name="AdditionalMnemonicEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Additional Mnemonic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_name">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_company">
     <item>
      <widget class="QLabel" name="label_company">
       <property name="text">
        <string>Company:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_company"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_mainMnemonic">
     <item>
      <widget class="QLabel" name="label_mainMnemonic">
       <property name="text">
        <string>Main Mnemonic:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_mainMnemonic"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_unit">
     <item>
      <widget class="QLabel" name="label_unit">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_unit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
